scripts/opt/backup-loop.sh

```diff
@@ -272,9 +278,17 @@ tar() {
     fi
   }
   prune() {
-    log INFO "Pruning backup files older than ${PRUNE_BACKUPS_DAYS} days"
-    if [ -n "$(_find_old_backups -print -quit)" ]; then
-      _find_old_backups -print -delete | awk '{ printf "Removing %s\n", $0 }' | log INFO
+
+    if [ -n "${PRUNE_BACKUPS_DAYS}" ] && [ "${PRUNE_BACKUPS_DAYS}" -gt 0 ]; then
+      log INFO "Pruning backup files older than ${PRUNE_BACKUPS_DAYS} days"
+      if [ -n "$(_find_old_backups -print -quit)" ]; then
+        _find_old_backups -print -delete | awk '{ printf "Removing %s\n", $0 }' | log INFO
+      fi
+    fi
+
+    if [ -n "${PRUNE_BACKUPS_COUNT}" ] && [ "${PRUNE_BACKUPS_COUNT}" -gt 0 ]; then
+      log INFO "Pruning backup files to keep only the latest ${PRUNE_BACKUPS_COUNT} backups"
+      _find_extra_backups | xargs -0 -I {} sh -c 'rm -v "{}"' | log INFO
```

Comment:
This part

```
xargs -0 -I {} sh -c 'rm -v "{}"'
```

seems overly complicated. Maybe it could just be

```suggestion
      _find_extra_backups | xargs -n 1 rm -v | log INFO
```